### PR TITLE
Allow setting a base URL

### PR DESCRIPTION
**Pull request by rowanmanning**
This allows us to run the Image Service on a sub-path rather than
immediately after the hostname.

### CODE REVIEW

File: lib/image-service.js [hunk 1]
```diff
@@ -110,3 +112,13 @@ function mountRoutes(app) {
 		resolve: initRoute => initRoute(app)
 	});
 }
+
+function sanitizeBaseUrl(url) {
```
Comment (JakeChampion) [inline lib/image-service.js]: Hmm, didn't think of that.

Comment (rowanmanning) [inline lib/image-service.js]: Alright to merge in that case?

**Comment by rowanmanning** [comment]
@JakeChampion actually after thinking about it on the way into work I think this solution is the best one to go with for now. I don't think it's a _huge_ amount of work to overhaul it, but this is fine for MVP. I've opened an issue to revisit this later (#54). Am I OK to merge?

**Comment by JakeChampion** [comment]
Yep :)

**Pull request merged**